include/dxc/Support/HLSLOptions.h

```diff
@@ -160,26 +161,8 @@ class DxcOpts {
 
   // SPIRV Change Starts
 #ifdef ENABLE_SPIRV_CODEGEN
-  bool GenSPIRV;                           // OPT_spirv
-  bool VkInvertY;                          // OPT_fvk_invert_y
-  bool VkInvertW;                          // OPT_fvk_use_dx_position_w
-  bool VkUseGlLayout;                      // OPT_fvk_use_gl_layout
-  bool VkUseDxLayout;                      // OPT_fvk_use_dx_layout
-  bool SpvEnableReflect;                   // OPT_fspv_reflect
-  bool VkNoWarnIgnoredFeatures;            // OPT_Wno_vk_ignored_features
-  bool SpvDebugFile;                       // OPT_fspv_debug
-  bool SpvDebugSource;                     // OPT_fspv_debug
-  bool SpvDebugLine;                       // OPT_fspv_debug
-  bool SpvDebugTool;                       // OPT_fspv_debug
-  llvm::StringRef VkStageIoOrder;          // OPT_fvk_stage_io_order
-  llvm::SmallVector<int32_t, 4> VkBShift;  // OPT_fvk_b_shift
-  llvm::SmallVector<int32_t, 4> VkTShift;  // OPT_fvk_t_shift
-  llvm::SmallVector<int32_t, 4> VkSShift;  // OPT_fvk_s_shift
-  llvm::SmallVector<int32_t, 4> VkUShift;  // OPT_fvk_u_shift
-  std::vector<std::string> VkBindRegister; // OPT_fvk_bind_register
-  llvm::SmallVector<llvm::StringRef, 4> SpvExtensions; // OPT_fspv_extension
-  llvm::StringRef SpvTargetEnv;                        // OPT_fspv_target_env
-  llvm::SmallVector<llvm::StringRef, 4> SpvOconfig;    // OPT_Oconfig
+  bool GenSPIRV;                    // OPT_spirv
+  SpirvCodeGenOptions SpirvOptions; // All SPIRV CodeGen-related options
```

Comment:
We should use `SPIR-V` for comments. That is the formal spelling.

Reply:
Done